controllers/alert_controller.go

```diff
@@ -18,13 +18,16 @@ package controllers
 
 import (
 	"context"
+	"strings"
 
 	"github.com/go-logr/logr"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
-	notificationv1alpha1 "github.com/fluxcd/notification-controller/api/v1alpha1"
+	apiv1 "github.com/fluxcd/notification-controller/api/v1alpha1"
```

Comment:
Given the fact that it is our own API, I think the `api` prefix is redundant (and not adding any additional context) and it can be just `v1`.